apis/duck/enqueue.go

```diff
@@ -33,12 +35,12 @@ type EnqueueInformerFactory struct {
 var _ InformerFactory = (*EnqueueInformerFactory)(nil)
 
 // Get implements InformerFactory.
-func (cif *EnqueueInformerFactory) Get(gvr schema.GroupVersionResource) (cache.SharedIndexInformer, cache.GenericLister, error) {
-	inf, lister, err := cif.Delegate.Get(gvr)
+func (cif *EnqueueInformerFactory) Get(ctx context.Context, gvr schema.GroupVersionResource) (cache.SharedIndexInformer, cache.GenericLister, error) {
+	inf, lister, err := cif.Delegate.Get(ctx, gvr)
 	if err != nil {
 		return nil, nil, err
 	}
-	// If there is an informer, attach our event handler.
+	// If there is an informer, attach our event ƒhandler.
```

Comment:
```suggestion
	// If there is an informer, attach our event handler.
```

Reply:
What is this? 💩 😃

Reply:
done in 77678bb83f220dd422bf886e7ce009f6f23d6616